### PR TITLE
test

### PR DESCRIPTION
**Pull request by sraikimaxime**

### CODE REVIEW

File: a.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+This is a test file
```
Comment (sraikimaxime) [inline a.txt]: I does look right !

Comment (amauryw) [inline a.txt]: :octocat: please amend

Comment (amauryw) [reply a.txt]: ✨ 🏗 test

Comment (commentordevelopers) [inline a.txt]: New comment here

Comment (amauryw) [reply a.txt]: 😨 nice

Comment (sraikimaxime) [reply a.txt]: test

**Pull request closed without merging**